pkg/virt-handler/virtwrap/manager.go

```diff
@@ -416,3 +427,125 @@ func (l *LibvirtDomainManager) KillVM(vm *v1.VM) error {
 	l.recorder.Event(vm, kubev1.EventTypeNormal, v1.Deleted.String(), "VM undefined")
 	return nil
 }
+
```

Comment:
I think that structure is good.  Also add virtwrap/domainmanager/common.go for any code that the drivers themselves need to share.

Having the two drivers in the same module means they can share non-public functions.  I'd rather it be possible to pull a driver out of tree, and thus each driver goes into its own module, and all common code be defined as a public interface.  For example, as structure like 
virtwrap/domainmanagers/
virtwrap/domainmanagers/common/
virtwrap/domainmanagers/libvirt/
virtwrap/domainmanagers/vdsm/

Would enforce that.  the code in /common/ would be Contract over time, and could live in its own repo...which it would have to do for out of tree drivers.

Just so you know what I am driving at.  Separate files is fine for now, we can enforce the rest via code review.